### DISCUSSION TITLE
Error "Git LFS is disabled for this repository."

### DISCUSSION THREAD

**Discussion by otaviokamel** [community/community#38436] [Actions]
Hello there,

I was trying to download some annotations for images in the following repository: https://github.com/eBay/modanet.

But when I tried with git bash with lfs, I got the following message:

Downloading annotations/modanet2018_instances_train.json (106 MB)
Error downloading object: annotations/modanet2018_instances_train.json (b8a3fd0): Smudge error: Error downloading annotations/modanet2018_instances_train.json (b8a3fd0dcfa8d6f209a0cef88cedd76decbf593bf47b4246504f681b7fb625af): batch response: Git LFS is disabled for this repository.

Errors logged to 'D:\pCloud\Matérias\CIn - Residência e Pós Gradução em Visão Computacional\TCC - Visão Computacional\TCC - Image Recommender Systems\modanet-master\modanet\.git\lfs\logs\20221106T165405.4879911.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: annotations/modanet2018_instances_train.json: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

I searched for this error online, and saw a solution that the user got this problem solved by asking direct support from GitHub: 
https://stackoverflow.com/questions/54209740/batch-response-git-lfs-is-disabled-for-this-repository

So, I'm writing here to see if someone can help me, because I'm not an advanced Git user.

Thanks, Otávio Kamel.

**Comment by david-engelmann** [comment]
It looks like the repository has been archived. The github support email address is support@github.com, it might take them a bit to respond. [You can also go to this url](https://support.github.com/contact) and select restore deleted data.

**Reply by otaviokamel** [reply]
> It looks like the repository has been archived. The github support email address is [support@github.com](mailto:support@github.com), it might take them a bit to respond. [You can also go to this url](https://support.github.com/contact) and select restore deleted data.

But can I restore a repository owned by another person?

**Reply by david-engelmann** [reply]
You can fork it and download your forked version

**Reply by otaviokamel** [reply]
I forked it, tried to download and got the same error.
I tried then looking for deleted repositories, but there is none.
I then deleted my fork of modanet repository, and tried to restore it, but it's under the message: "These repositories were deleted, but can't be restored by you. Contact support if you want to restore them."

**Comment by david-engelmann** [comment]
On your forked version of the repository, enabled LFS again. It’s not clear why it was disabled, but you should be good to go now. Please let me know if you have any further issue with this.

**Comment by otaviokamel** [comment]
I might be doing something wrong, or maybe I'm not understanding yours instructions, but here's what I tried now:

I checked this option in my forked repository
![image](https://user-images.githubusercontent.com/33270911/200584954-21024014-0c52-46a7-8d76-51ad8ef26002.png)

I tried cloning it
![image](https://user-images.githubusercontent.com/33270911/200585037-4a95063f-1a7a-4b01-8d95-22357a54dc1c.png)

I tried fetching it
![image](https://user-images.githubusercontent.com/33270911/200585099-01597bfa-9b07-45a1-b42b-36096098cf2f.png)

Are my steps right/wrong? I'm not sure why the error persists.

**Comment by david-engelmann** [comment]
You'll need to enable it in the repo's setting, you can [follow this guide](https://docs.github.com/en/github-ae@latest/admin/user-management/managing-repositories-in-your-enterprise/configuring-git-large-file-storage-for-your-enterprise) to enable LFS for a repository.
